### PR TITLE
Add recipe for drms

### PR DESCRIPTION
**Pull request by SolarDrew**
Recipe for [drms](https://github.com/kbg/drms). @kbg, I've put you as a maintainer, as it's your package, but I can take you off if you don't want the extra work.

### CODE REVIEW

**Comment by Cadair** [comment]
Can someone review this please? Thanks!

File: recipes/drms/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 1
```
Comment (isuruf) [inline recipes/drms/meta.yaml]: Set this to `0`

File: recipes/drms/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.9.0
+    - pandas >=0.14.1
+    - six >=1.8.0
```
Comment (isuruf) [inline recipes/drms/meta.yaml]: `numpy`, `pandas`, `six` can be dropped here.

File: recipes/drms/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+test:
+  imports:
+    - drms
+
+about:
+  home: https://github.com/kbg/drms
+  license: MIT
+  license_family: MIT
+  license_file: ''
```
Comment (isuruf) [inline recipes/drms/meta.yaml]: Add the path to the license file here

File: recipes/drms/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  description: |
+    The ``drms`` module provides an easy-to-use interface for accessing HMI, AIA and MDI data with Python.
+    It uses the publicly accessible `JSOC <http://jsoc.stanford.edu/>`_ DRMS server by default, but can also\nbe used with local `NetDRMS <http://jsoc.stanford.edu/netdrms/>`_ sites.
+    More information, including a detailed tutorial is available on `Read the Docs <http://drms.readthedocs.io/>`_.
+  doc_url: http://drms.readthedocs.io/
+  dev_url: https://github.com/kgb/drms
+
+extra:
+  recipe-maintainers:
+    - kbg
```
Comment (isuruf) [inline recipes/drms/meta.yaml]: Is `kbg` willing to maintain? If not, please remove the name

Comment (Cadair) [reply recipes/drms/meta.yaml]: @SolarDrew can you add me and @nabobalis while you are at it? thanks.

Comment (SolarDrew) [reply recipes/drms/meta.yaml]: I've not heard anything from him, so I'll remove it. No problem @Cadair, I'll do that.

File: recipes/drms/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    It uses the publicly accessible `JSOC <http://jsoc.stanford.edu/>`_ DRMS server by default, but can also\nbe used with local `NetDRMS <http://jsoc.stanford.edu/netdrms/>`_ sites.
+    More information, including a detailed tutorial is available on `Read the Docs <http://drms.readthedocs.io/>`_.
+  doc_url: http://drms.readthedocs.io/
+  dev_url: https://github.com/kgb/drms
+
+extra:
+  recipe-maintainers:
+    - SolarDrew
+    - Cadair
+    - nabobalis
```
Comment (isuruf) [inline recipes/drms/meta.yaml]: Now the question is whether @nabobalis is okay with this

File: recipes/drms/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: 'LICENSE.txt'
+  summary: Access HMI, AIA and MDI data with Python
+  description: |
+    The ``drms`` module provides an easy-to-use interface for accessing HMI, AIA and MDI data with Python.
+    It uses the publicly accessible `JSOC <http://jsoc.stanford.edu/>`_ DRMS server by default, but can also\nbe used with local `NetDRMS <http://jsoc.stanford.edu/netdrms/>`_ sites.
+    More information, including a detailed tutorial is available on `Read the Docs <http://drms.readthedocs.io/>`_.
+  doc_url: http://drms.readthedocs.io/
+  dev_url: https://github.com/kgb/drms
```
Comment (isuruf) [inline recipes/drms/meta.yaml]: `kgb` -> `kbg`

File: recipes/drms/meta.yaml (hunk 5 above)
Comment (SolarDrew) [reply recipes/drms/meta.yaml]: I assume so if Cadair says to add him - @nabobalis speak now or forever etc.

Comment (SolarDrew) [reply recipes/drms/meta.yaml]: I've just spoken to him, he's ok with it 👍

Comment (nabobalis) [reply recipes/drms/meta.yaml]: Yes

**Comment by SolarDrew** [comment]
Thanks for your comments @isuruf :)

File: recipes/drms/meta.yaml (hunk 4 above)
Comment (kbg) [reply recipes/drms/meta.yaml]: Sorry @SolarDrew, I must have forgotten to write a reply. I'm ok with (co-)maintaining the recipe, so feel free to keep me as a maintainer.

Comment (SolarDrew) [reply recipes/drms/meta.yaml]: No problem, I'll put you back on the list :)

**Comment by mwcraig** [comment]
Thanks, everyone, merging!

**Comment by Cadair** [comment]
thanks @mwcraig!

**Pull request merged**